helix-term/src/commands/typed.rs

```diff
@@ -1819,7 +1819,15 @@ fn update_goto_line_number_preview(cx: &mut compositor::Context, args: Args) ->
 
     let scrolloff = cx.editor.config().scrolloff;
     let line = args[0].parse::<usize>()?;
-    goto_line_without_jumplist(cx.editor, NonZeroUsize::new(line));
+    goto_line_without_jumplist(
+        cx.editor,
+        NonZeroUsize::new(line),
+        if cx.editor.mode == Mode::Select {
+            Movement::Extend
+        } else {
+            Movement::Move
+        },
```

Comment:
Not sure what to do there since I made `goto_line_without_jumplist` take the `Movement`.

Reply:
I think it makes sense to check the mode here like this. It will be the same behavior as before this change at least